src/App.jsx

```diff
@@ -1 +1,11 @@
 
+function App() {
+  return (
+    <div className="App">
+      <TodoList />
+    </div>
+  );
```

Comment:
Are you using this className `App`? 
Because I noticed you don't have any `css` files, if you don't need the class, it should be removed.

By the way, are you applying any styles? If not I would recommend at least reusing the styling from the React todo app you did already, but not ignoring styles